### PR TITLE
[WIP] upload image to IPFS

### PR DESCRIPTION
**Pull request by divergio**
Still having IPFS import issues:

The requested module '/node_modules/merge-options/index.js?v=5a4a1ecd' does not provide an export named 'default'
SyntaxError: The requested module '/node_modules/merge-options/index.js?v=5a4a1ecd' does not provide an export named 'default'

### CODE REVIEW

File: src/routes/mint/[address]/[id].svelte [hunk 1]
```diff
@@ -1,17 +1,79 @@
 <script lang="ts">
 	import { page } from '$app/stores';
 	import { onMount } from 'svelte';
-	import * as IPFS from 'ipfs-core';
+	import { tokenUriToMetaData } from '$lib/nft-utils';
+	import { evaluateNft, getTokenUriType, getImageType } from '$lib/utils/functions';
+	import defaultAbi from '$lib/defaultAbi';
+	import { user } from '$lib/stores/user';
+	import { create } from 'ipfs-http-client'
```
Comment (divergio) [inline src/routes/mint/[address]/[id].svelte:8]: @ryanlabouve still having import issues, can you try to get a minimal ipfs-http-client working? 

import { create } from 'ipfs-http-client'

const client = create('https://ipfs.infura.io:5001/api/v0');

let results = client.add("a string that will live on IPFS"

Comment (ryanlabouve) [reply src/routes/mint/[address]/[id].svelte:8]: I think `ipfs-http-client` is the node package.

Try using the library from ipfs-core like `import { create } from 'ipfs-core'`

ref: https://github.com/ipfs-examples/js-ipfs-examples/blob/master/examples/browser-add-readable-stream/src/index.js#L3

Comment (ryanlabouve) [reply src/routes/mint/[address]/[id].svelte:8]: Side stepping with https://github.com/ryanlabouve/nftrusty-exploration/pull/28 for now

**Comment by divergio** [comment]
see #35

**Pull request closed without merging**